web/src/components/SpanDetail/components/HTTPAttributesTabs.tsx

```diff
@@ -0,0 +1,77 @@
+import {Table, Tabs, Typography} from 'antd';
+import React from 'react';
+import styled from 'styled-components';
+import CustomTable from '../../CustomTable';
+
+const STabs = styled(Tabs)`
+  .ant-tabs-content-holder {
+    overflow: hidden;
+    padding-bottom: 100px;
+  }
+`;
+
+interface IProps {
+  attributeList: any[];
+}
+
+export const HTTPAttributesTabs = ({attributeList}: IProps): JSX.Element => {
+  const render = (value: any) => <span data-cy="assertion-check-property">{value}</span>;
+  return (
+    <>
+      <Typography.Title level={5} style={{margin: 0}}>
+        Attributes
+      </Typography.Title>
+      <STabs style={{marginBottom: 150, overflow: 'hidden'}}>
+        <Tabs.TabPane style={{overflow: 'hidden'}} tab={<span>Request</span>} key="span-request-detail">
+          <CustomTable
+            size="small"
+            pagination={{hideOnSinglePage: true}}
+            dataSource={attributeList?.filter(a => {
+              return (
```

Comment:
I think we should do this either at the model or at the service level. Let's not have business logic at the component level

Reply:
done